### ISSUE TITLE
Complex Analysis prerequisities - missing Alg Top 1?

### ISSUE THREAD

**Issue by msm-code** [vEnhance/napkin#245]
This is a remark about the Complex Analysis section.

Chapters 32 (Meromorphic functions) and 33 (Holomorphic square roots) directly reference chapter 66 (Covering projections) and 65 (Fundamental Groups). For example:

![image](https://github.com/user-attachments/assets/6dbccac2-dee7-4a1c-9737-c121bd3bf892)

"Punctured plane" and π1 are introduced in chapter 65, and Theorem 66.2.5 is obviously introduced in chapter 66. As a learner, I find it hard to follow some arguments without this context. I think it's warranted to add a soft prerequisite between the Algebraic Topology 1 section and the Complex Analysis section.

In don't feel confident enough to make a PR for it myself. But in case you consider it reasonable, it's enough to add: 

```
\coreqc  23,10,64,30;0:    % AT1 -> Cmplx Ana
```

to tex/frontmatter/digraph.tex. You might want to tweak the layout a little though:

![image](https://github.com/user-attachments/assets/4862db85-999a-4e55-8d77-06947ebfa360)

**Comment by vEnhance** [comment]
Makes sense, I've added that in as a dotted arrow. Thanks for suggestion.
